README.md

```diff
@@ -112,11 +112,19 @@ Currently the project is being primarily co-ordinated via the GDG Melbourne slac
 
 ### Pull Request Process
 
+## Method 1 - If you are part of the team
 - Develop your work in a branch. Branch names should be prefixed with `feature`, `tech`, or `bugfix` depending on what is being done in them. eg: `feature\update-readme`
 - Open a pull request into master. Be sure to include a detailed description in your pull request. Other contributors will comment on your code or approve it
 - Please be sure to note the build status of your PR reported by the Codemagic CI
 - Once your pull request has been approved, please merge into master.
 
+## Method 2 - If you are not yet part of the team
+- If you are not a part of the team, you can still work on the code by clicking on the `Fork` option in this page.
+- Once you have forked it, you would be able to clone it and work on it.
+- To create a pull request, you would have to push your local changes back to your git account. 
+- You would then find the option to `Compare and Pull Request`. 
+- Please follow the instructions above on how to submit a pull request. 
+
```

Comment:
Can you put something here about approvals?  What actually happens?  Are you able to merge or does somebody on the team have to do it?

Reply:
I think someone on the team would have to merge it as I am not able to find the option to do it. Will update the readme with this!